examples/videocomposer/vc/utils/pt2ms.py

```diff
@@ -132,3 +133,62 @@ def load_pt_weights_in_model(model, checkpoint_file_pt, state_dict_refiners=None
         dst_path = f"{src_path}.bak"
         print(f"moving {src_path} to {dst_path} ...")
         shutil.move(src_path, dst_path)
+
+    path = "model_weights/non_ema_141000_no_watermark.npy"
```

Comment:
Where is the npy from? Try to allow this two-step conversion (pth -> npy -> ckpt) to execute in one command.

Please update readme to give illustartion.

Reply:
done